Makefile

```diff
@@ -1,7 +1,44 @@
-BUILD_DIR=./build
+SHELL := /bin/bash
 
-clean:
-	rm -rf ${BUILD_DIR}
+# Env var inputs for container builds
+# IMAGE: The name of the image to build, regardless of the registry and the tag.  Any image
+# 		 will be built for docker hub (docker.io) and github (ghcr.io) container registry.
```

Comment:
This doesn't seem correct, it will only be built for `${REGISTRY}`

Reply:
I'm sorry but I don't see which comments are duplicated, can you elaborate?